quesma/quesma/ui/console_routes.go

```diff
@@ -182,7 +235,28 @@ func (qmc *QuesmaManagementConsole) initPprof(router *mux.Router) {
 	router.HandleFunc("/debug/pprof/trace", pprof.Trace)
 }
 
+var store = sessions.NewCookieStore([]byte("test"))
+
+func authMiddleware(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		//if !authEnabled {
+		//	next.ServeHTTP(w, r)
+		//	return
+		//}
+		session, err := store.Get(r, quesmaSessionName)
+		userID, ok := session.Values["userID"].(string)
+		if !ok || userID == "" || err != nil {
```

Comment:
It will be tricky to guess "why it redirects me to the login form". We should log and error here.

Reply:
I'd expect this to be pretty usual behavior, but agree that `logger.Warn().Msgf("User not authenticated, redirecting to login page")` is worth having there